### PR TITLE
[WIP] Feature verification verifier

### PR DESCRIPTION
**Pull request by rwrzesien**
Resolves https://github.com/golemfactory/concent/issues/220
Resolves https://github.com/golemfactory/concent/issues/221
Resolves https://github.com/golemfactory/concent/issues/225
Resolves https://github.com/golemfactory/concent/issues/226

### CODE REVIEW

File: concent_api/concent_api/system_check.py [hunk 1]
```diff
@@ -93,6 +93,75 @@ def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: di
     return []
 
 
+@register()
+def check_settings_storage_server_internal_address(app_configs, **kwargs):  # pylint: disable=unused-argument
```
Comment (dybi) [inline concent_api/concent_api/system_check.py]: 1) please add unit tests for this check
2) please extract error to a fixture/function, like `def create_error_13_ssl_cert_path_is_none():`

File: concent_api/concent_api/system_check.py [hunk 2]
```diff
@@ -93,6 +93,75 @@ def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: di
...
+                'STORAGE_SERVER_INTERNAL_ADDRESS is not a valid URL',
+                hint = '{}'.format(error),
+                id   = 'concent.E011',
+            )]
+
+    return []
+
+
+@register()
+def check_settings_verifier_storage_path(app_configs, **kwargs):  # pylint: disable=unused-argument
```
Comment (dybi) [inline concent_api/concent_api/system_check.py]: 1) please add unit tests for this check
2) please extract error to a fixture/function, like `def create_error_13_ssl_cert_path_is_none():`

File: concent_api/concent_api/system_check.py [hunk 3]
```diff
@@ -93,6 +93,75 @@ def check_settings_storage_cluster_address(app_configs, **kwargs):  # pylint: di
...
+                'Cannot write to VERIFIER_STORAGE_PATH',
+                hint = 'Current user does not have write permissions to directory {}'.format(settings.VERIFIER_STORAGE_PATH),
+                id   = 'concent.E017',
+            )]
+
+    return []
+
+
+@register()
+def check_settings_blender_max_rendering_time(app_configs, **kwargs):  # pylint: disable=unused-argument
```
Comment (dybi) [inline concent_api/concent_api/system_check.py]: 1) please add unit tests for this check
2) please extract error to a fixture/function, like `def create_error_13_ssl_cert_path_is_none():`

**Comment by rwrzesien** [comment]
Closing as this will be split into smaller pull requests.

**Pull request closed without merging**